### PR TITLE
Log more details for command execution failures

### PR DESCRIPTION
**Pull request by nan-yu**
https://oss.gprow.dev/view/gs/oss-prow-build-kpt-config-sync/logs/multi-repo-1-autopilot-regular/1582649977673355264 failed to get cluster credentials. We need the stdout to understand what happened.

### CODE REVIEW

File: e2e/nomostest/ntopts/gke.go [hunk 1]
```diff
@@ -56,25 +56,25 @@ func GKECluster(t testing.NTB, apiServerTimeout time.Duration) Opt {
 func forceAuthRefresh(t testing.NTB) {
 	out, err := exec.Command("kubectl", "config", "current-context").CombinedOutput()
 	if err != nil {
-		t.Fatalf("failed to get current config: %v", err)
+		t.Fatalf("failed to get current config (stdout: %s): %v", string(out), err)
```
Comment (sdowell) [inline e2e/nomostest/ntopts/gke.go:59]: `CombinedOutput` returns both stdout and stderr, so might be clearer to mention both. I also expect the stdout/stderr could be verbose, so I'd suggest putting it at the end of the string. e.g.
```
t.Fatalf("failed to get current config: %v\nstdout/stderr:\n%s", err, string(out))
```

Comment (nan-yu) [reply e2e/nomostest/ntopts/gke.go]: done

File: e2e/nomostest/ntopts/gke.go [hunk 2]
```diff
@@ -56,25 +56,25 @@ func GKECluster(t testing.NTB, apiServerTimeout time.Duration) Opt {
...
 	gkeProject := gkeArgs[1]
 	gkeZone := gkeArgs[2]
 	gkeCluster := gkeArgs[3]
 
-	_, err = exec.Command("gcloud", "container", "clusters", "get-credentials",
+	out, err = exec.Command("gcloud", "container", "clusters", "get-credentials",
 		gkeCluster, "--zone", gkeZone, "--project", gkeProject).CombinedOutput()
 	if err != nil {
-		t.Fatalf("failed to get credentials: %v", err)
+		t.Fatalf("failed to get credentials %v\nstdout/stderr:\n%s", err, string(out))
```
Comment (sdowell) [inline e2e/nomostest/ntopts/gke.go:73]: minor nit: I liked having the `:` here and above :)

/lgtm
/approve

Comment (nan-yu) [reply e2e/nomostest/ntopts/gke.go]: done

**Pull request merged**